### PR TITLE
[staging-next] dev86: unstable-2022-07-19 -> unstable-2024-03-28

### PR DESCRIPTION
**Pull request by ophestra**
Fixes build failure.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/by-name/de/dev86/package.nix [hunk 1]
```diff
@@ -1,17 +1,18 @@
 { lib
 , stdenv
-, fetchFromGitHub
+, fetchFromGitea
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "dev86";
-  version = "unstable-2022-07-19";
+  version = "unstable-2024-03-28";
```
Comment (AndersonTorres) [inline pkgs/by-name/de/dev86/package.nix:8]:
```suggestion
  version = "0-unstable-2024-03-28";
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/by-name/de/dev86/package.nix (hunk 1 above)
Comment (ophestra) [reply pkgs/by-name/de/dev86/package.nix]: Done.

**Review by AndersonTorres — APPROVED** [review]

**Comment by AndersonTorres** [comment]
It is failing by some reason...

**Comment by ophestra** [comment]
Why is this still failing? It seems to be failing somewhere completely unrelated.

**Review by SigmaSquadron — APPROVED** [review]
> Why is this still failing? It seems to be failing somewhere completely unrelated.

Someone seems to have broken `staging`. Try rebasing to the latest `staging` commit.

The changes themselves look good to me.

**Comment by FliegendeWurst** [comment]
This should go in staging-next or master as it fixes the build.

**Comment by SigmaSquadron** [comment]
> This should go in staging-next or master as it fixes the build.

As a committer, you should be able to change the branch through the web interface.

**Review by SigmaSquadron — APPROVED** [review]

**Comment by FliegendeWurst** [comment]
> > This should go in staging-next or master as it fixes the build.
> 
> As a committer, you should be able to change the branch through the web interface.

Yes, but a rebase is needed anyhow (previous staging changes are not in master or staging-next)

**Comment by FliegendeWurst** [comment]
Done, just remembered that I can push to PRs where the author allowed it.

File: pkgs/by-name/de/dev86/package.nix [hunk 2]
```diff
@@ -1,17 +1,18 @@
 { lib
 , stdenv
-, fetchFromGitHub
+, fetchFromGitea
 }:
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "dev86";
-  version = "unstable-2022-07-19";
+  version = "0-unstable-2024-03-28";
```
[the remark this answers is not in the document]
Comment (vcunat) [reply pkgs/by-name/de/dev86/package.nix:8]: Opened as https://github.com/NixOS/nixpkgs/pull/366951

**Comment by SigmaSquadron** [comment]
Superseded by #366951

**Pull request closed without merging**